### PR TITLE
net-misc/ethercard-diag: update HOMEPAGE, #552094

### PR DESCRIPTION
**Pull request by wimmuskee**
Gentoo-Bug: https://bugs.gentoo.org/552094

### CODE REVIEW

File: net-misc/ethercard-diag/ethercard-diag-20050321.1g.ebuild [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 # $Id$
 
 inherit toolchain-funcs
 
 DESCRIPTION="low level mii diagnostic tools including mii-diag and etherwake (merge of netdiag/isa-diag)"
-HOMEPAGE="ftp://ftp.scyld.com/pub/diag/ ftp://ftp.scyld.com/pub/isa-diag/"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
 SRC_URI="mirror://gentoo/${P}.tar.lzma"
 
 LICENSE="GPL-2"
```
Comment (SoapGentoo) [inline net-misc/ethercard-diag/ethercard-diag-20050321.1g.ebuild]: in `src_compile`, change `emake || die` -> `default`
change `src_install` to:
```
src_install() {
	default
	dodir /sbin
	mv "${ED%/}"/usr/sbin/{mii-diag,ether-wake} "${ED%/}"/sbin/ || die
}
```

Comment (wimmuskee) [inline net-misc/ethercard-diag/ethercard-diag-20050321.1g.ebuild]: I'm assuming I should update the EAPI's for both to 6 as well.

Comment (SoapGentoo) [inline net-misc/ethercard-diag/ethercard-diag-20050321.1g.ebuild]: If possible, yes

Comment (wimmuskee) [inline net-misc/ethercard-diag/ethercard-diag-20050321.1g.ebuild]: added the requested changes

**Pull request closed without merging**